packages/content/src/collections/docs.ts

```diff
@@ -0,0 +1,40 @@
+import { defineCollection } from "@content-collections/core";
+import { transformMDX } from "@fumadocs/content-collections/configuration";
+
+import { mdxOptions } from "@/collections/utils.js";
+
+/*
+ * Collection for `/docs` documentation pages.
+ *
+ * Note that there are no `Api/` docs in this collection; API docs are stored in
+ * `apiDocs` collection.
+ *
+ * The key reason for this is that API docs have a different frontmatter schema
+ * (no `title` field; `sections` list). This means that we have to inject API
+ * docs in the website's `source.ts` config, though. Maybe we should inject them
+ * here in `transform` instead.
```

Comment:
This comment above and the code in `website/src/app/source.ts` (configuration of sources for fumadocs) describe a decision I had to make for how we assemble the docs and inject `/docs/Api`.

I'm still not sure if I should've done it another way. I wish content-collections supported per-document schema somehow...